src/utils/getTokensChainData.ts

```diff
@@ -44,7 +44,7 @@ const getTokensChainData = async (listName: string, addressArray?: string[], cha
       symbol: tokenInfoResponse?.[i * 3]?.[0] ?? "",
       address,
       chainId,
-      decimals: tokenInfoResponse[i * 3 + 2][0],
+      decimals: tokenInfoResponse?.[i * 3 + 2]?.[0] ?? 18,
```

Comment:
can we add a todo here? We shouldn't set default decimal. Should filter those invalid results instead

Reply:
ok got it